windows/cavendish_particle_tracks.bat

```diff
@@ -1,9 +1,8 @@
-:: This script will start napari with the Cavendish Particle Tracks plugin
-:: Set the seed for shuffling the images (CPT_SHUFFLING_SEED) to a different value in each lab computer
-:: Create a shortcut to this file and place it in the Public Desktop folder
-:: If you are feeling creative, you can also change the icon of the shortcut to the Cavendish logo (C:\Program Files\Python39\Lib\site-packages\cavendish_particle_tracks\docs\ParticleCrocodile.png)
+:: This is an example batch file to run the python script start_cpt.py
+:: The file start_cpt.py is a Python script that starts napari with the Cavendish Particle Tracks plugin
```

Comment:
```suggestion
:: This is an example batch file to run the python script start_cpt.py
:: The file start_cpt.py is a Python script that starts napari with the Cavendish Particle Tracks plugin
:: If you are feeling lab-patriotic, you can also change the icon of the shortcut to the Cavendish logo (https://github.com/palvarezc/cavendish-particle-tracks/tree/main/windows/CPT.ico)
```

Oh! I see! How about giving them that instruction? Surely they can download it along with these other Windows files.